nav2_map_server/params/vector_object_server_params.yaml

```diff
@@ -0,0 +1,31 @@
+vector_object_server:
+  ros__parameters:
+    map_topic: "vo_map"
+    global_frame_id: "map"
+    resolution: 0.05
+    default_value: -1
+    overlay_type: 0
+    update_frequency: 1.0
+    transform_tolerance: 0.1
+    shapes: ["Poly", "CircleA", "CircleB"]
+    Poly:
+      type: "polygon"
+      frame_id: "map"
+      closed: True
+      value: 100
+      points: [0.3, 0.3, 0.3, -0.3, 0.0, -0.3, 0.0, 0.3]
+    CircleA:
+      type: "circle"
+      frame_id: "map"
+      fill: True
+      value: 10
+      center: [3.0, 3.0]
+      radius: 0.5
+      uuid: "7b3f3d7d-135c-4b6c-aca1-7a84d1050505"
```

Comment:
I didn't think about this before, but if we're asking users to specify shapes in a params file, UUID might not be the best option. That would require them to offline generate some UUIDs to put into the params file. If we instead just made it an `id` field of `unsigned int` then it would be easier to enumerate them in a config file.

When its generated via code, UUID makes sense, but not when manually requiring annotation

Reply:
BTW, `uuid` ROS-param is optional, it could be skipped by developer: in this case it will be generated to a new shape automatically. In this case, does it still have a sense to have an ID?

I'm asking, because when we have an ID, we do not need to support UUID anymore: any shape could be uniquely addressed by its ID, it also could be easily generated if not specified w/o any UUID dependency. So we could just skip it.

What do you think, maybe it is better to totally refuse from UUID usage in favor of ID?
Another question that logically raises here: how about to use ID of string type, which could be called "name"? It is even much more user-friendly, than unsigned ints, I believe; and we do not need to worry about ID-s/UUID-s in ROS-parameters: name of each shape is already specified in its parameters path and `shapes` array.

Reply:
However, from my point of view, using UUID is still the nice way for UX:
- We could optionally specify it in ROS-parameters
- We could optionally specify it in `AddShapes.srv` service request when adding a new shape
- And we to specify it on mandatory basis when modifying existing shape while calling `AddShapes.srv` and removing some specific shape in `RemoveShape.srv`

Reply:
>  uuid ROS-param is optional, it could be skipped by developer

Ah, ok! I'd just make sure to document that in the param guide that its optional except if you might want to remove it, in which case specifying it is important.